src/components/DateRangePicker.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { DateRangePicker as DefaultDateRangePicker, IDateRangePickerProps } from '@blueprintjs/datetime/lib/esm/dateRangePicker'
+import '@blueprintjs/datetime/lib/css/blueprint-datetime.css'
+import styled from 'styled-components'
+
+const StyledDatePicker = styled(DefaultDateRangePicker)`
+  justify-content: center;
+
+  & div {
```

Comment:
Can we make this selector more precise?
Is there a class name (`& .class`)?
Is it a direct child (`& > div`)?
Or does the parent have a class name (`& .class > div`)?